documentation/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Troubleshooting Kubernetes vSphere Cloud Provider
+---
+To troubleshoot issues with Kuberenetes cluster deployment, please visit [https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/)
+
+## Logs to inspect
+To debug vSphere Cloud Provider Issues, Controller-Manager, API-server and Kubelet logs should be checked.
```

Comment:
This should be a bullet list as follows :
Following logs are needed to debug VCP issues -
- Controller Manager logs
- API-server logs
- kubelet logs